bio_page.html

```diff
@@ -0,0 +1,99 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <script src="https://code.jquery.com/jquery-3.5.1.min.js"></script>
+    <link href="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/css/bootstrap.min.css" rel="stylesheet">
+    <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/js/bootstrap.min.js"></script>
+    <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/js/bootstrap.bundle.min.js"></script>
+    <link rel="stylesheet" type="text/css" href="style.css">
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Bio Page</title>
+</head>
+<script>
+    window.addEventListener("load", function () { alert("Hello!! Welcome to my Bio page!!") })
+</script>
+<nav class="navbar navbar-expand-lg navbar-dark" style="background-color:rgb(152, 152, 152)">
+    <a class="navbar-brand" href="index.html">Home</a>
+    <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarNavAltMarkup"
+        aria-controls="navbarNavAltMarkup" aria-expanded="false" aria-label="Toggle navigation">
+        <span class="navbar-toggler-icon"></span>
+    </button>
+    <div class="collapse navbar-collapse" id="navbarNavAltMarkup">
+        <div class="navbar-nav">
+            <a class="nav-item nav-link" href="bio_page.html">Bio Page<span class="sr-only"></span></a>
+            <a class="nav-item nav-link" href="portfolio.html">Portfolio</a>
+            <a class="nav-item nav-link" href="experience.html">Experience Page</a>
+            <a id='view_socials' class="nav-link dropdown-toggle" href="#" id="navbarDropdownMenuLink" role="button"
+                data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+                View Socials
+            </a>
+            <div id='socials' class="dropdown-menu" aria-labelledby="navbarDropdownMenuLink">
+                <li><a href="https://www.linkedin.com/in/eliascruz25/">LinkedIn</a></li>
+                <li><a href="https://github.com/eli2412">GitHub</a></li>
+            </div>
+</nav>
+<script>
+    $(document).ready(function () {
```

Comment:
oooo jquery! as a stretch goal, is there a way to adjust this to perform similar functionality without using jquery?